### PR TITLE
added blog post and diagram

### PR DESCRIPTION
**Pull request by stockholmux**
*Description of changes:*
Adding technical blog post 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by stockholmux** [comment]
FYI - I have additional reviewers off github on this blog post. Please hold on merge - I will likely have additional changes now.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    ` }
+}
+```
+
+
+No matter what happens, we don’t want our customers to be able see anything in the `private` object. We’ll use Open Distro’s built-in access control functionality to make this happen.
+
+## Creating a user & role
+
+First up in Kibana we need to create a user. This user will be used by the application for serving public information, so we’ll call it `public-item-user`. First, go to the “Security” item from the side menu then click “Internal users” in the secondary menu. Now, click on the “Create internal user” button. This will take you to `<yourhost>:<yourport>/app/opendistro_security#/users/create`.
```
Comment (aetter) [inline _posts/2020-10-23-fine-grained-access-control-for-search.md:40]: I generally recommend **Bold** for any GUI item that you have to click, but whatever you think is appropriate.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                "price" : 5.99
+            }
+        }
+    ]
+}
+```
+
+Notice that the `private` object is gone. Our user and role has filtered this data right out.
+
+One additional note here - we setup these two roles to *only* have access to this particular index. That means that if,  by some mistake or happenstance the credentials were released for these users or an application bug somehow allowed for passing in other indices, you’re covered. No other index managed by Open Distro would be at risk of disclosure as access control would deny the query. You can think of this as an implementation of the principle of least privilege.
```
Comment (aetter) [inline _posts/2020-10-23-fine-grained-access-control-for-search.md:84]: The setup process (noun) is one word, whereas the verb is set up.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+To setup a route for the employee section of your application, you would create a role called `private-item-viewer`  that is allowed to see all information. Repeat the same steps as above except skipping the step where you exclude `private` in the “Field level security section”. Then you would create a user called `private-item-user` with the role of `private-item-viewer`. All of the requests from to backend for employees would uses this user to make the requests, enabling them to have access to all the fields in the object `private`.
```
Comment (aetter) [inline _posts/2020-10-23-fine-grained-access-control-for-search.md:86]: I think you can convert this to present tense and just make the whole thing read a bit more cleanly.

To set up a route for the employee section of your application, create a role called `private-item-viewer` that has permissions to see all information. Repeat the process above, except skip the step where you exclude the `private` field. Then create a user...

Etc. etc.

**Review by aetter — APPROVED** [review]
I like the post a lot. I'd never thought about access control from this perspective.

I don't know if we have access to a copy editor for these posts, but if we do, I think it'd be valuable to have someone give it a once-over.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+
+```
+PUT ecommerce-items/_doc/brown-mug
+{
+    "title": "Brown Mug",
+    "description": "This mug features a large handle and a marbled, brown ceramic design.",
+    "price" : 5.99,
+    "private" : {
+    `    "cost" : 1,
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:29]: Those are just code block and inline monospace formatting.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md (hunk 1 above)
Comment (stockholmux) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:40]: I'll change that.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+layout: posts
+author: Kyle Davis
+comments: true
+title: "Using fine grained access control for search"
+categories:
+- technical-posts
+feature_image: "https://d2908q01vomqb2.cloudfront.net/ca3512f4dfa95a03169c5a670a4c91a19b3077b4/2019/03/26/open_disto-elasticsearch-logo-800x400.jpg"
+---
+
+Open Distro for Elasticsearch has extensive access control capabilities built right in. Of course, access control can be used to keep people out of sensitive information, it can also be used to simplify building applications that implement search with Open Distro for Elasticsearch. Let’s explore this a little more.
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:11]: Fixed. Good catch.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md (hunk 1 above)
Comment (stockholmux) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:40]: Perfect. I'll use bold for clickables, double quotes for headings and labels, and monospaced for type-ins.

File: _posts/2020-10-23-fine-grained-access-control-for-search.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (aetter) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:29]: No, I mean the stray backticks within the code block. Code block formatting is already monospace, and inline monospace can't span lines as far as I know, so it renders out with broken JSON like this:

backticks

[the remark this answers is not in the document]
Comment (stockholmux) [reply _posts/2020-10-23-fine-grained-access-control-for-search.md:29]: oh weird. I exported this from quip. I'll make sure this doesn't get into the revision.

**Pull request merged**